src/azure-cli/azure/cli/command_modules/sql/commands.py

```diff
@@ -732,24 +749,6 @@ def load_command_table(self, _):
             is_preview=True)
         g.wait_command('wait')
 
-    with self.command_group('sql midb log-replay',
```

Comment:
May I know the reason why removing these commands?
If you don't want to support them any more, please follow deprecation policy.